ivy/functional/backends/paddle/layers.py

```diff
@@ -58,7 +58,23 @@ def conv1d(
     dilations: Union[int, Tuple[int]] = 1,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    if data_format == "NWC":
+        x = paddle.transpose(x, perm=(0, 2, 1))
+    x = _pad_before_conv(x, filters, strides, padding, 1, dilations, data_format)
+    filters = paddle.transpose(filters, perm=(2, 1, 0))
+    if not isinstance(padding, str):
+        padding = "VALID"
+    res = paddle.nn.functional.conv3d(
```

Comment:
you are using conv3d instead of conv1d, is that intentional?

Reply:
My bad... It should be conv1d! Do you want me to change and recommit them?